bin/get-google-analytics-pageviews

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env ruby
+require "csv"
+require "google/apis/analyticsreporting_v4"
+require "googleauth"
+require "yaml"
+
+GA_REPORTING = Google::Apis::AnalyticsreportingV4
+PAGEVIEWS = "ga:dimension2"
+RENDERING_APP = "ga:dimension20"
+
+def load_config_data
+  YAML.load(File.open("config/ga_env.yml")).each do |key, value|
+    ENV[key.to_s] = value
+  end
+end
+
+def matrix_file
+  Dir["**"].select { |f| f =~ /\Amatrix_\d{4}-\d{2}-\d{2}_\d{2}:\d{2}:\d{2}.csv/ }.sort.last
+end
+
+def get_google_analytics_views(document_type, rendering_app)
+  return 0 unless document_type && rendering_app
```

Comment:
Does this mean that `0` will end up in the spreadsheet? I wonder if it's better just to leave this blank since 0 could be a valid number of page views.